### ISSUE TITLE
a json file loaded as a js file in npm:diffie-hellman

### ISSUE THREAD

**Issue by faassen** [jspm/jspm-cli#444]
I'm trying to use the "jsonld" library with jspm. I install it and then do a:

```
import jsonld from 'jsonld';
```

when I look at the console output, I see these errors:

```
GET http://localhost:5000/jspm_packages/npm/diffie-hellman@3.0.1/lib/primes.js 404 (Not Found)
es6-module-loader.src.js:139 Potentially unhandled rejection [3] Error loading "npm:diffie-hellman@3.0.1/lib/primes" at http://localhost:5000/jspm_packages/npm/diffie-hellman@3.0.1/lib/primes.js
Error loading "npm:diffie-hellman@3.0.1/lib/primes" from "app/main" at http://localhost:5000/lib/main.js
Not Found: http://localhost:5000/jspm_packages/npm/diffie-hellman@3.0.1/lib/primes.js (WARNING: non-Error used)
```

and indeed, there is no ``primes.js`` in that directory. There is however a ``primes.json``, that somehow is being accessed as a ``.js`` file.

I think this blocks the whole loading process, causing jsonld not to be imported and my .js code not to be executed.

**Comment by guybedford** [comment]
Thanks for reporting - I've created https://github.com/jspm/npm/issues/43 which will fix this.

I'll aim to squeeze this into the release tomorrow.

In the mean time, I would suggest manually editing `jspm_packages/npm/diffie-hellman@3.0.1/browser.js` to `require('./lib/primes.json!')` and running `jspm install json`.

**Comment by guybedford** [comment]
@faassen this has been resolved in https://github.com/jspm/npm/issues/43.

I tested this again, and now it's throwing that `fs.readFileSync` is not defined. Unfortunately this is a browserify limitation for the same library, so you should chase up the browserify support with them.

**Comment by guybedford** [comment]
This is fixed in jspm@0.11.0.
